config/targets.json

```diff
@@ -68,5 +68,15 @@
   {
     "name": "federalist",
     "slack_channel": "federalist"
+  },
+  {
+    "name": "compliance-toolkit",
```

Comment:
Blergh, Team API is [still broken](https://github.com/18F/team-api.18f.gov/issues/271), so [the name change](https://github.com/18F/concourse-compliance-testing/pull/88) won't propagate. 👎